install_ffmpeg.sh

```diff
@@ -111,7 +111,12 @@ else
   if which clang > /dev/null; then
     echo "clang detected, building with GPU support"
 
-    EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid --enable-filter=scale_cuda --enable-encoder=h264_nvenc --enable-libtensorflow --enable-filter=lvpdnn"
+    EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid --enable-filter=scale_cuda --enable-encoder=h264_nvenc"
+
+    if [[ $BUILD_TAGS == *"experimental"* ]]; then
```

Comment:
Oh cool does the `*...*` syntax result in a substring check for "experimental" in `BUILD_TAGS`?

Reply:
Yep!